<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShortcutSettingsWidget</class>
 <widget class="QWidget" name="ShortcutSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="actionsGroupBox">
     <property name="title">
      <string>Keyboard shortcuts</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="filterLineEdit">
        <property name="placeholderText">
         <string>Filter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTreeWidget" name="actionsTreeWidget">
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="resetAllHorizontalLayout">
        <item>
         <spacer name="resetAllHorizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>388</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="resetAllPushButton">
          <property name="text">
           <string>Reset all</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="shortcutGroupBox">
     <property name="title">
      <string>Shortcut</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="keySequenceHorizontalLayout">
        <item>
         <widget class="QLabel" name="keySequenceLabel">
          <property name="text">
           <string>Key sequence:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="keySequenceLineEdit">
          <property name="placeholderText">
           <string>Enter key sequence as text</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="buttonsHorizontalLayout">
        <item>
         <spacer name="buttonsHorizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>308</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="ShortcutButton" name="recordPushButton">
          <property name="text">
           <string>Record</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="resetPushButton">
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="warningLabel">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;Key sequence has potential conflicts.&lt;/span&gt;&lt;a href=&quot;#conflicts&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;Show.&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ShortcutButton</class>
   <extends>QPushButton</extends>
   <header location="global">dialogs/shortcut_settings/ShortcutButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
